### ISSUE TITLE
Code completion does not work in the middle of a module name

### ISSUE THREAD

**Issue by ericglau** [ericglau/cairo-ls#32]
Press Ctrl-Space in the middle of a module name.  The code completion suggestions empty.

![image](https://user-images.githubusercontent.com/29640130/169561927-4889a943-7f63-4420-aadf-a8631971ee06.png)

Currently it only works if you press Ctrl-Space immediately after `from `.

![image](https://user-images.githubusercontent.com/29640130/169562364-aa0cd102-87a3-454d-814f-afec38203938.png)

**Comment by cwkang1998** [comment]
Would love to contribute, may I take this issue up?

**Comment by ericglau** [comment]
@cwkang1998 Yes of course, that would be great!  See [project setup steps](https://github.com/ericglau/cairo-ls/blob/main/CONTRIBUTING.md) and let me know if you run into any issues.  The part of the code for completions is [here](https://github.com/ericglau/cairo-ls/blob/6bb0500d770c8708cb071b1d6ff4152ae5f53cb8/server/src/server.ts#L1040).
